com.unity.render-pipelines.high-definition/Runtime/Lighting/LightLoop/GlobalLightLoopSettings.cs

```diff
@@ -114,18 +115,22 @@ public enum CubeCookieResolution
     }
 
     /// <summary>
-    /// Possible values for one element of the density volume atlas.
+    /// Possible values for one element of the Local Volumetric Fog atlas.
     /// </summary>
     [Serializable]
-    public enum DensityVolumeResolution
+    public enum LocalVolumetricFogResolution
```

Comment:
Am I wrong or this is not renamed by the script upgrader anymore?
At least keep the former and tag it obsolete.

Reply:
No need because it was added on the same version as this changes,

https://github.com/Unity-Technologies/Graphics/pull/2751